src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static String getRandomColor() {
+        Color[] colors = Color.values();
+        int randomIndex = random.nextInt(colors.length);
+        return colors[randomIndex].toString();
```

Comment:
```suggestion
        return colors[randomIndex].name();
```
read common mistakes